### PR TITLE
(DPS-2069) Change the stable version of API docs

### PR DESCRIPTION
**Pull request by ale-kacperczyk**
## 🚀 Links

- [Feature branch](https://developers.labs.livechat.com/docs/feature/dps-2069)
- [Jira](https://livechatinc.atlassian.net/browse/DPS-2069)

## 📓 Description

- Created directories for all v3.1 docs and manually moved the files there. v3.2 docs are now the default files (not in any directory).
- Adjusted #Versioning descriptions in each doc. 
- Added new const `LEGACY_VERSION_1` for v3.1.

### Content

- New content
  
### Features (for the website)

- New feature

### CODE REVIEW

File: src/constant/index.js [hunk 1]
```diff
@@ -1,9 +1,10 @@
 export const VERSIONS_GROUPS = {
   DEFAULT: {
-    STABLE_VERSION: "3.1",
+    STABLE_VERSION: "3.2",
     LEGACY_VERSION: "2.0",
```
Comment (jakubsikora) [inline src/constant/index.js:4]:
```suggestion
    LEGACY_VERSIONS: ["2.0", "3.1"],
```

See other comments related to it

File: src/components/core/version.js [hunk 2]
```diff
@@ -178,7 +183,7 @@ const Version = ({ articleVersions, redirectToVersion }) => {
         }
       >
         <DesktopNote>
-          {selectedVersion === versions.LEGACY_VERSION && (
+          {(selectedVersion === versions.LEGACY_VERSION || selectedVersion === versions.LEGACY_VERSION_1) && (
```
Comment (jakubsikora) [inline src/components/core/version.js:186]:
```suggestion
          {(versions.LEGACY_VERSIONS.includes(selectedVersion)) && (
```

File: src/components/core/version.js [hunk 3]
```diff
@@ -149,9 +154,9 @@ const Version = ({ articleVersions, redirectToVersion }) => {
         <span style={{ marginLeft: "3px" }}>
           {version === versions.STABLE_VERSION
             ? `(stable)`
-            : version === versions.LEGACY_VERSION
-            ? `(legacy)`
-            : `(dev preview)`}
+            : version === versions.LEGACY_VERSION || version === versions.LEGACY_VERSION_1
```
Comment (jakubsikora) [inline src/components/core/version.js:157]:
```suggestion
            : (versions.LEGACY_VERSIONS.includes(version))
```

File: src/components/core/version.js [hunk 4]
```diff
@@ -108,6 +108,11 @@ const Warning = ({ selectedVersion, versionColor, versions }) => (
           API. For the rest, refer to v3.1.
         </p>
       )}
+      {selectedVersion === versions.LEGACY_VERSION_1 && (
```
Comment (jakubsikora) [inline src/components/core/version.js:111]: See below, you could use something similar with `includes`

File: src/utils/index.js [hunk 5]
```diff
@@ -56,7 +56,7 @@ export const versionToString = number =>
 
 export const getVersionColor = (version, groupVersions) => {
   const isStable = version === groupVersions.STABLE_VERSION;
-  const isLegacy = version === groupVersions.LEGACY_VERSION;
+  const isLegacy = (version === groupVersions.LEGACY_VERSION || version === groupVersions.LEGACY_VERSION_1);
```
Comment (jakubsikora) [inline src/utils/index.js:59]:
```suggestion
  const isLegacy = groupVersions.LEGACY_VERSIONS.includes(version);
```

File: src/components/core/version.js (hunk 4 above)
Comment (ale-kacperczyk) [reply src/components/core/version.js]: Discussed to do it differently

**Pull request merged**